asterius/rts/rts.gc.mjs

```diff
@@ -244,9 +314,16 @@ export class GC {
           }
         }
       } else {
+        // If the closure belongs to the static part of the memory,
```

Comment:
Indeed we avoided future checks, but imo the real point here is to enable the code below to push the static closure to the scavenging work list. We don't evacuate static objects, but we do scavenge them.